src/pages/Trade/components/TradePairOrder/index.tsx

```diff
@@ -17,14 +18,55 @@ export default () => {
   const [visiblePopver, setVisiblePopver] = useState(false)
   const { price, setPrice: priceOnChange, pay, setPay: payOnChange, receive, setStep } = Order
   const formRef = React.createRef<FormInstance>()
-  const [disabled] = useState(false)
   const [buyer, seller] = Order.pair
+  const [disabled, setDisabled] = useState(false)
+  const [submitText, setSubmitText] = useState(i18n.t(`trade.placeOrder`))
 
   const changePair = () => {
     setVisiblePopver(false)
     Order.togglePair()
     form.resetFields()
   }
+  // dai -> ckb 卖单
+  const checkPay = (_: any, value: string): Promise<void> => {
```

Comment:
If the first variable has useless, please remove it.

Reply:
it's fixed

Reply:
I mean `_: any`

Reply:
no, it's necessary. this is antd form paramater required.

Reply:
no , it's mecessary paramter  require.

Reply:
OK. I think it's very strange